ocs_ci/utility/oc_utils.py

```diff
@@ -0,0 +1,52 @@
+import logging
+
+from semantic_version import Version
+
+from ocs_ci.utility.utils import exec_cmd
+
+log = logging.getLogger(__name__)
+
+
+def get_client_version():
+    """
+    Get the major.minor version of the openshift client
+
+    Returns:
+        str: major.minor oc version
+
+    """
+    # get version info from oc
+    cmd = "oc version"
```

Comment:
What about using `oc version -o json` or `oc version -o yaml`, wouldn't that be slightly nicer for processing and maybe more stable format (you will not rely on the order of the output lines etc.)? I see, that processing of the plain `oc version` is quite easy... so maybe it is not worth that.
```json
$ oc version -o json
{
  "clientVersion": {
    "major": "",
    "minor": "",
    "gitVersion": "openshift-clients-4.6.0-202006250705.p0-168-g02c110006",
    "gitCommit": "02c110006bfef4ba53fa5042bb9eae170dd3dc1c",
    "gitTreeState": "clean",
    "buildDate": "2020-12-16T08:02:28Z",
    "goVersion": "go1.15.2",
    "compiler": "gc",
    "platform": "linux/amd64"
  },
  "serverVersion": {
    "major": "1",
    "minor": "20",
    "gitVersion": "v1.20.0+bafe72f",
    "gitCommit": "bafe72fb05eddc8246040b9945ec242b9f805935",
    "gitTreeState": "clean",
    "buildDate": "2021-03-14T16:01:39Z",
    "goVersion": "go1.15.7",
    "compiler": "gc",
    "platform": "linux/amd64"
  },
  "releaseClientVersion": "4.6.9",
  "openshiftVersion": "4.7.0-0.nightly-2021-03-25-035231"
}
```

Reply:
I am now using `-o json`